backend/hexa/workspaces/schema/types.py

```diff
@@ -177,6 +180,18 @@ def resolve_workspace_pipeline_tags(workspace: Workspace, info, **kwargs):
     )
 
 
+@workspace_membership_object.field("organizationMembership")
+def resolve_workspace_membership_organization_membership(
+    membership: WorkspaceMembership, info, **kwargs
+):
+    """Return the user's organization membership if the workspace belongs to an organization."""
+    if membership.workspace.organization:
+        return OrganizationMembership.objects.filter(
+            user=membership.user, organization=membership.workspace.organization
+        ).first()
```

Comment:
Any reason you choose `filter().first()` here over `get()`? I'd say if there this query returns multiple results, it would be good to get a Sentry notification.